addons/csw/stringtable.xml

```diff
@@ -184,8 +184,9 @@
             <Spanish>Almacenamiento de munición</Spanish>
         </Key>
         <Key ID="STR_ACE_CSW_handleExtraMagazinesType_description">
-            <English>Determines whether extra magazines are stored on the ground or inside an ammo box</English>
-            <German>Legt fest, ob zusätzliche Magazine auf dem Boden oder in einer Munitionskiste aufbewahrt werden</German>
+            <English>Determines whether extra magazines are stored on the ground or inside an ammo box, if the static weapon doesn't have a defined container.</English>
```

Comment:
I don't know what the standard is across ACE, but I think that if you change the english meaning you should clear out all the other untranslated ones. @jonpas comments?

Reply:
Correct.

Reply:
I can invalidate the old strings but in https://github.com/acemod/ACE3/pull/9240 we opted to not do so because it was a minor change not being worth getting rid of translations. I figured this was about the same level of change.

Reply:
This is commenting on new functionality, not elaborating on existing. That is my basis for the purge